CHANGELOG/CHANGELOG-1.5.0.md

```diff
@@ -48,3 +67,5 @@
 - CSI-Powerstore: Update the upgrade procedure. ([#484](https://github.com/dell/csm/issues/484))
 - step_error: command not found in karavi-observability-install.sh. ([#479](https://github.com/dell/csm/issues/479))
 - "repctl cluster inject --use-sa" doesn't work for Kubernetes 1.24 and above. ([#463](https://github.com/dell/csm/issues/463))
+- Update CSM Observability release note with github issue link and support matrix. ([#455](https://github.com/dell/csm/issues/455))
```

Comment:
#455 if for v1.4.0 rather than v1.5.0. I removed the tag v1.5.0 from #455. 
FYI, @forrestxia .

Reply:
Hi @shaynafinocchiaro , can we delete line #70? `Update CSM Observability release note with github issue link and support matrix. ([#455](https://github.com/dell/csm/issues/455))`

Reply:
I don't see https://github.com/dell/csm/issues/519 in the bug list

Reply:
Hi @baoy1, the bug has been removed.

Reply:
Hi @YianZong, 519 is not visible in the PR but I do see it under [Bugs](https://github.com/dell/csm/blob/changelog-staging/CHANGELOG/CHANGELOG-1.5.0.md#bugs)

Reply:
Ok, I see, this is an incremental. Thank you, @shaynafinocchiaro